lib/src/commands/OptionsProcessor.ts

```diff
@@ -134,4 +136,31 @@ export class OptionsProcessor {
       options[key].passProps = undefined;
     }
   }
+
+  private processSearchBar(key: string, value: any, options: Record<string, any>) {
```

Comment:
This is great! I'd like to suggest that we handle all deprecated options in a dedicated private method, which we'll be able to delete in the next major version when these options become obsolete. It's a cosmetic change which will increase readability and ease when upgrading RNN.

WDYT?

Reply:
As above^ :D